hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/amrmproxy/TestFederationInterceptor.java

```diff
@@ -590,6 +593,10 @@ public Object run() throws Exception {
         interceptor.recover(recoveredDataMap);
 
         Assert.assertEquals(1, interceptor.getUnmanagedAMPoolSize());
+
+        // Waiting for SC-1 to time out.
+        Thread.sleep(800);
```

Comment:
Can we just wait for it to be 1?

Reply:
Thank you for your help to review the code! I will modify the code as soon as possible.